billing-service/src/main/java/com/petclinic/billing/datalayer/DataSetupService.java

```diff
@@ -22,11 +22,11 @@ public DataSetupService(BillService billService) {
     public void run(String... args) throws Exception {
 
 
-        BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2021,9,19),59.99,BillStatus.PAID);
-        BillRequestDTO b3 = new BillRequestDTO( "3", "operation", "1", LocalDate.of(2021,9,21), 199.99,BillStatus.PAID);
-        BillRequestDTO b4 = new BillRequestDTO( "4", "injury", "1",  LocalDate.of(2021,9,22), 199.99,BillStatus.UNPAID);
-        BillRequestDTO b2 = new BillRequestDTO( "2", "operation", "2", LocalDate.of(2021,9,20), 199.99,BillStatus.OVERDUE);
-        BillRequestDTO b5 = new BillRequestDTO( "5", "chronic", "3",  LocalDate.of(2021,9,23), 199.99,BillStatus.UNPAID);
+        BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2023,9,19),59.99,BillStatus.PAID, LocalDate.of(2023, 10,3));
+        BillRequestDTO b3 = new BillRequestDTO( "3", "operation", "1", LocalDate.of(2023,9,21), 199.99,BillStatus.PAID,LocalDate.of(2023, 10,5));
+        BillRequestDTO b4 = new BillRequestDTO( "4", "injury", "1",  LocalDate.of(2023,9,30), 199.99,BillStatus.UNPAID,LocalDate.of(2023, 10,10));
```

Comment:
I think this might be a small error here as the due date and bill date are not 14 days apart

Reply:
Thank you for letting me know even though it's a small error. it's fixed and they are now 14 days apart like others.